x-pack/test_serverless/functional/test_suites/observability/cases/configure.ts

```diff
@@ -21,23 +21,30 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
   const find = getService('find');
 
   describe('Configure Case', function () {
-    //  Error: timed out waiting for assertRadioGroupValue: Expected the radio group value to equal "close-by-pushing"
-    this.tags(['failsOnMKI']);
```

Comment:
@cnasikas did you actually verify that this is no longer failing when running against MKI? The PR CI won't tell you that...

Reply:
For the records: details have been shared in slack.

Reply:
@pheyos I tested and the test is working as expected. I noticed another error though and I decided to keep it skipped. The error:

`// security_exception: action [indices:data/write/delete/byquery] is unauthorized for user [elastic] with effective roles [superuser] on restricted indices [.kibana_alerting_cases], this action is granted by the index privileges [delete,write,all]`

I will open an issue to address it. I can see it in other cases too.

Reply:
https://github.com/elastic/kibana/issues/168809